src/utils/restUtils.ts

```diff
@@ -13,22 +12,22 @@ export function erQ(): boolean {
 
 export function erDevGcp(): boolean {
     const url = window.location.href;
-    return (url.indexOf(".dev-nav.no") > 0);
+    return url.indexOf(".dev-nav.no") > 0;
 }
 
 export function erLabsGcp(): boolean {
     const url = window.location.href;
-    return (url.indexOf("..labs.nais.io") > 0);
+    return url.indexOf("..labs.nais.io") > 0;
 }
 
 export function erMockServer(): boolean {
     const url = window.location.origin;
-    return (url.indexOf("digisos-test") > 0) || (url.indexOf("dev-nav.no") > 0) || (url.indexOf("labs.nais.io") > 0);
+    return url.indexOf("digisos-test") > 0 || url.indexOf("dev-nav.no") > 0 || url.indexOf("labs.nais.io") > 0;
```

Comment:
Er det mulig å la prettier beholde disse?

Reply:
Jeg har ikke sett så mye på hva som kan konfigureres av Prettier. Meningen her er vel at den tar bort paranteser som er unødvendige. Er de nødvendige for logikken blir de værende.

I tilfellet her hadde det kanskje vært mer lesbart hvis vi brukte `startsWith` eller `includes` i stedet for `indexOf`? :)

Reply:
`startsWith()` og `includes()` fungerer dessverre ikke i Internet Explorer 😢 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes

Er enig i at man burde få ha paranteser der det gjør koden mer lesbar.

Reply:
Både `startsWith()` og `includes()` skal være dekket av core-js tho, så jeg kan jo se om vi kan skrive om slike ting etterhvert. :) https://github.com/zloirock/core-js#ecmascript-string-and-regexp

Jeg fant heller ikke noe i prettier-docen om å beholde paranteser der prettier mener det er unødvendig.